domain/src/main/java/org/oppia/domain/profile/ProfileTestHelper.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.domain.profile
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.oppia.app.model.ProfileId
+import javax.inject.Inject
+
+class ProfileTestHelper @Inject constructor(
```

Comment:
Please add KDoc for this helper to explain when it should be used &  how.

Reply:
Added KDoc.